sdk/python/pyproject.toml

```diff
@@ -1,10 +1,9 @@
 [build-system]
-requires = ["hatchling"]
+requires = ["hatchling", "hatch-vcs"]
 build-backend = "hatchling.build"
 
 [project]
-name = "dagger"
-version = "0.1.0"
+name = "dagger-io"
```

Comment:
is CI failing because of this? (can't find `dagger` module)

Reply:
Yes, it is. Hatch is using the normalized named in `site-packages` so I need to figure out how to work around it.

Reply:
What should matter is that what's inside of `src/dagger`.

Reply:
having same issue with generating API docs:

```
# in dagger/sdk/python/
mkdir docs
sphinx-apidoc -F --implicit-namespace -o docs/ src/
make -C docs html
```

```
...
WARNING: autodoc: failed to import module 'dagger.server.cli' from module 'src'; the following exception was raised:
No module named 'dagger'
WARNING: autodoc: failed to import module 'dagger.server.converter' from module 'src'; the following exception was raised:
No module named 'dagger'
WARNING: autodoc: failed to import module 'dagger.server' from module 'src'; the following exception was raised:
No module named 'dagger'
```